### DISCUSSION TITLE
How about fixing bug of reordering icons

### DISCUSSION THREAD

**Discussion by juliycesar** [valinet/ExplorerPatcher#3804] [General]
It is pity that Microsoft still have this bug. Here related conversation. 
https://answers.microsoft.com/en-us/windows/forum/all/why-do-taskbar-buttons-change-order-when-switching/a2c81dee-6d42-4ef1-b400-ed827e6bf73e

It is very handy feature when you set up own order of windows for instance console first , then some editor, then chrome and switch it with Win+1,2,3.

But after switching to different virtual desktop it reorders it and win+1 shows different program.

**Comment by pyrates999** [comment]
That is beyond what EP is for.  EP can only enable code that has been disabled.

**Comment by Amrsatrio** [comment]
Use Windhawk for this. Next version of EP will support Windhawk mods in its taskbar.
